modules/oadp-release-notes-1-1-4.adoc

```diff
@@ -0,0 +1,47 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/oadp-release-notes.adoc
+
+:_content-type: REFERENCE
+[id="migration-oadp-release-notes-1-1-4_{context}"]
+= OADP 1.1.4 release notes
+
+The OADP 1.1.4 release notes lists any new features, resolved issues and bugs, and known issues.
+
+[id="new-features1.1.4_{context}"]
+== New features
+
+This version of OADP is a service release. No new features are added to this version.
+
+[id="resolved-issues1.1.4_{context}"]
+== Fixed bugs
+
+The following bugs have been fixed in this release:
+
+* link:https://issues.redhat.com/browse/OADP-1557[OADP-1557]
+* link:https://issues.redhat.com/browse/OADP-1822[OADP-1822]
+* link:https://issues.redhat.com/browse/OADP-1511[OADP-1511]
+* link:https://issues.redhat.com/browse/OADP-1642[OADP-1642]
+* link:https://issues.redhat.com/browse/OADP-1398[OADP-1398]
+* link:https://issues.redhat.com/browse/OADP-1267[OADP-1267]
+* link:https://issues.redhat.com/browse/OADP-1390[OADP-1390]
+* link:https://issues.redhat.com/browse/OADP-1650[OADP-1650]
+* link:https://issues.redhat.com/browse/OADP-1487[OADP-1487]
+
+
+[id="known-issues1.1.4_{context}"]
+== Known issues
+
+This release has the following known issues:
+
+* OADP backups might fail because a UID/GID range might have changed on the cluster where the application has been restored, with the result that OADP does not back up and restore OpenShift UID/GID range metadata.
```

Comment:
All of the lines need to be be collapsed into a single paragraph for each of the list items.
```suggestion
* OADP backups might fail because a UID/GID range might have changed on the cluster where the application has been restored, with the result that OADP does not back up and restore {product-title} UID/GID range metadata.
```